sympy/functions/special/tests/test_bessel.py

```diff
@@ -757,3 +757,7 @@ def test_marcumq():
 
     x = Symbol('x', integer=True)
     assert marcumq(x, a, a).rewrite(besseli) == marcumq(x, a, a)
+
+def test_issue_26134():
+    x = Symbol('x')
+    assert marcumq(2, 3, 4).rewrite(Integral, x=x).dummy_eq(Integral(x**2*exp(-x**2/2 - Rational(9, 2))*besseli(1, 3*x), (x, 4, oo))/3)
```

Comment:
There should be two blank lines between top-level functions and long lines should be wrapped or broken into multiple lines by introducing an intermediate variable.